config/jobs/kubernetes-sigs/kind/kind-release-blocking.yaml

```diff
@@ -49,6 +48,13 @@ periodics:
           # this is mostly for building kubernetes
           memory: 9Gi
           cpu: 7
+    nodeSelector:
+      cloud.google.com/machine-family: c4
+    tolerations:
+      - key: "dedicated"
+        operator: "Equal"
+        value: "sig-testing"
+        effect: "NoSchedule"
```

Comment:
Just for my understanding: is this a permanent thing or a temporary stop-gap?

Reply:
In previous PRs I did put a comment on these, but we will want to mass remove the toleration from _all_ jobs when the taint is removed from the node pool and the old node pool is phased out.

Similarly we'll want to remove the node selector at the same time

Reply:
thanks!